lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_codefreeze_prechecks.rb

```diff
@@ -14,18 +14,21 @@ def self.run(params)
         # Checkout develop and update
         Fastlane::Helper::GitHelper.checkout_and_pull('develop')
 
+        app = ENV['PROJECT_NAME'].nil? ? params[:app] : ENV['PROJECT_NAME']
+
         # Create versions
-        current_version = Fastlane::Helper::Android::VersionHelper.get_release_version
-        current_alpha_version = Fastlane::Helper::Android::VersionHelper.get_alpha_version
+        current_version = Fastlane::Helper::Android::VersionHelper.get_release_version(app)
+        current_alpha_version = Fastlane::Helper::Android::VersionHelper.get_alpha_version(app)
         next_version = Fastlane::Helper::Android::VersionHelper.calc_next_release_version(current_version, current_alpha_version)
-        next_alpha_version = ENV['HAS_ALPHA_VERSION'].nil? ? nil : Fastlane::Helper::Android::VersionHelper.calc_next_alpha_version(next_version, current_alpha_version)
+        next_alpha_version = current_alpha_version.nil? ? nil : Fastlane::Helper::Android::VersionHelper.calc_next_alpha_version(next_version, current_alpha_version)
```

Comment:
It seems that after your changes, there are only 3 instances of `HAS_ALPHA_VERSION` remaining in the codebase
```
lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb
54:          section = ENV['HAS_ALPHA_VERSION'].nil? ? 'defaultConfig' : 'vanilla {'
100:        #                or `nil` if `$HAS_ALPHA_VERSION` is not defined.
107:          return nil if ENV['HAS_ALPHA_VERSION'].nil?
335:          self.update_version(new_version_beta, ENV['HAS_ALPHA_VERSION'].nil? ? 'defaultConfig' : 'vanilla {')
```

It seems worth eliminating the remaining ones and add a mention in the CHANGELOG that host apps adopting this new version of the toolkit could get rid of it in their Fastfile

Reply:
I'll make mention of it in the Changelog that this file can be removed but at the moment it looks like the remaining ones are currently only used for backward compatibility. It allows for updating to this version and later versions without necessarily supporting the version.properties changes right away. Once all our clients have supported it we can definitely go ahead and remove all the instances.

Reply:
Ok make sense, we can totally handle this later (could be worth opening an issue, similar to [the one we have to get rid of `PROJECT_ROOT_FOLDER`)](https://github.com/wordpress-mobile/release-toolkit/issues/205), to not forget later)

Reply:
Created https://github.com/wordpress-mobile/release-toolkit/issues/277